Automattic/wp-calypso
# FSE: Remove custom publish button and panel, and replace with info notice

#### Changes proposed in this Pull Request

* Remove the custom publish button and warning panel for FSE template part updating
* Add a fixed info notice to warn users of the implications of template part update

Partly implements #34728
Fixes #34705

#### Testing instructions

* Check out this branch and rebuild the FSE plugin for your local FSE test environment `npx lerna run dev --scope='@automattic/full-site-editing'`
* Edit a Header or Footer template part and check that you see a blue warning notice about updates affecting multiple parts of the site
* The Publish button should also now follow the normal editor flows of 'Update' rather than always showing 'Publish' and there should be not warning panel on update.
* Test in both full screen mode and also normal mode and on different screen sizes.

Currently the notice is not fixed so scrolls up - given it is fixed I quite like this approach as it is not constantly stealing screen space if user scrolls to edit - but happy to hear thoughts on whether to fix it or not:

![notice](https://user-images.githubusercontent.com/3629020/62170756-aeb82400-b380-11e9-9454-be39c7cfe22a.gif)

This is how it looks with a stacked editor notice:

![stacked](https://user-images.githubusercontent.com/3629020/62170767-bd064000-b380-11e9-9958-5d1be2e2db7a.gif)
